### PR TITLE
Added sound & mainscene/credit scene

### PR DESCRIPTION
**Pull request by sejinkay**
Sound script for the moleclick isn't perfect (it plays when they spawn/respawn rather than when they get clicked on) because somehow when there's 'Invoke' in the script, sound code doesn't work.. I still added a sound for respawn though.
I also put some placeholder images/sounds for now. 
+ oh ignore the bush (just added to see how it works with other images than squares)

### CODE REVIEW

File: Guacamole/Assets/Scripts/GameControlScript.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 public class GameControlScript : MonoBehaviour
 {
-    public GameObject life1, life2, life3, gameover;
+    public GameObject life1, life2, life3;
```
Comment (GuiHeurich) [inline Guacamole/Assets/Scripts/GameControlScript.cs:8]: Why is gameover removed?

Comment (GuiHeurich) [reply Guacamole/Assets/Scripts/GameControlScript.cs:8]: I'm assuming it's moved somewhere else,  but I can't really find it.

Comment (sejinkay) [reply Guacamole/Assets/Scripts/GameControlScript.cs:8]: it's because gameover text is now on the gameover scene, not on the playscene.

**Review by GuiHeurich — COMMENTED** [review]
Amazing work!

**Pull request merged**